### ISSUE TITLE
Most Creative Lightning Talk 

### ISSUE THREAD

**Issue by PaulSpence** [adele157/easterlies-collaborative-project#71]
Hey kids. I want us to win the Most Creative Lightning Talk at the COSIMA workshop. Please help me re-write the words to "Its a long way to the top" by ACDC to match the science of our paper:

https://www.youtube.com/watch?v=vj_rvLVpqg8&ab_channel=AC%2FDC-Topic 

Lyrics:

… Ridin' down the highway
Goin' to a show
Stop in all the byways
Playin' rock 'n' roll
Gettin' robbed
Gettin' stoned
Gettin' beat up
Broken-boned
Gettin' had
Gettin' took
I tell you, folks
It's harder than it looks

… It's a long way to the top if you wanna rock 'n' roll
It's a long way to the top if you wanna rock 'n' roll
… If you think it's easy doin' one night stands
Try playin' in a rock roll band
It's a long way to the top if you wanna rock 'n' roll

**Comment by PaulSpence** [comment]
Blowin' down the coastline
Goin under shelves
Movin' all the sea ice
Playin' water flows

Gettin' blown
Gettin' warmed
Gettin' advected
Broken' homes ..?

help!

**Comment by navidcy** [comment]
Yo I wanna help out with this. I hear that your collaborators are not collaborative on this.

**Comment by PaulSpence** [comment]
Please do!

**Comment by PaulSpence** [comment]
I was practicing it last night ... pretty rockin :)

**Comment by PaulSpence** [comment]
can you play bag pipes?

**Comment by navidcy** [comment]
No. I can sing and help w lyrics. I can try bagpipes but won’t come out good. Will give it a go.

**Comment by navidcy** [comment]
This is a descent lyrics attempt. Will it be karaoke-d? What are you envisioning?

**Comment by navidcy** [comment]
Perhaps will rhyme better with

- Playin' downslope flows --> Goin' down the slope

Who else is in this band? We need rehearsing...!

**Comment by PaulSpence** [comment]
Playin' downslope flows --> Goin' down the slope

Nice edit!
